docs/customizing_your_tooltip.md

```diff
@@ -66,6 +66,7 @@ var options =
 | `onMove`        | function(event, item) | Callback when tooltip moves (e.g., when mouse moves within a bar). |
 | `onDisappear`   | function(event, item) | Callback when tooltip disappears (when mouse moves out an item). |
 | `colorTheme`    | String         | A color theme picker. <br>__Supported values:__ `"light"` and `"dark"`. <br>__Default value:__ `"light"` <br>To further customize, overwrite the `.vg-tooltip` class in your CSS. |
+| `sort`          | String | function | Sort fields on 'title', 'value', or pass a custom sort(a, b) function. |
```

Comment:
What does sort fields by "value" mean?  Do you mean sort tooltip "datum" table by the values, not the key?  
If that's the case, wouldn't that makes the table quite unstable as different data points would have totally different order? 

Please feel free to correct me if I misunderstood and perhaps add more explanation. :)

Reply:
This PR follows on #155, to provide more control over how the tooltip displays values for faceted line charts representing aggregated+grouped values.

Here's an illustration of the end result:
![vega-group-by-tooltips-2](https://user-images.githubusercontent.com/1127259/36635514-3f1533dc-196b-11e8-8d6d-b703b7c7f0bf.gif)

Note that when the line for China crosses above the line for US, the tooltip fields switch order as well. For this example, I passed `sort: 'value'`. (Note for context: I'm driving the tooltip with `rect` columns with `fillOpacity: 0`, each of which aggregates the values for each line series at that x-coordinate.)

To answer your other questions:

- Number sorts descending to match the visual ordering in a chart with values increasing further up the y-axis; Date and String sort ascending because that's how we usually order words and dates. However, if you feel it should be different, I'm happy to change, and can pass a custom `sort` function for my needs.

- `'value'` refers to the value of the tooltip field; the pair to `'title'`. I can refine this explanation in the README. And yes, as you can see in the illustration above, the intent is to change the ordering of fields in the tooltip to match the ordering of their corresponding marks.